pkgs/tools/networking/iodine/default.nix

```
@@ -2,20 +2,20 @@
 
 stdenv.mkDerivation rec {
   pname = "iodine";
-  version = "unstable-2019-09-27";
+  version = "v0.8.0";
 
   src = fetchFromGitHub {
     owner = "yarrick";
     repo = "iodine";
-    rev = "8e14f18";
-    sha256 = "0k8m99qfjd5n6n56jnq85y7q8h2i2b8yw6ba0kxsz4jyx97lavg3";
+    rev = "607b9d3";
```

We must use a long hash to prevent 404 when more commits are added by eg forks.

---

```suggestion
    rev = "v${version}";
```
actually we can use tags here